README.md

```diff
@@ -165,52 +166,71 @@ Confirm that the data has indeed been migrated by checking pgAdmin 4.
 
 - Export database snapshot: `pg_dump -h localhost -U USERNAME whiteclaw -f EXPORT-FILENAME.sql`
 - Wipe the existing database:
-   - Enter postgres interactively as the `postgres` user: `psql -U postgres -h localhost -W`
-   - Drop the database: `drop database whiteclaw;`
-   - Create the database: `create database whiteclaw;`
+  - Enter postgres interactively as the `postgres` user: `psql -U postgres -h localhost -W`
```

Comment:
we should probs add instructions for downloading / setting up Docker (and maybe Docker Desktop) since we have it for Postgres right now

Reply:
agreed^

Reply:
yes will do! why don't we do this while I help you guys setup everything so that we can document any of the struggles we go thru as well

Reply:
https://airtable.com/tbli2lWQHAqBa6ZcI/viwWvxb27XGuziw4c/reckto7XUbkv7odac?blocks=hide --> airtable ticket for this

Reply:
yes I like it let's do that